site/constants/index.ts

```diff
@@ -0,0 +1 @@
+export const FEEDBACK_LINK = "/feedback";
```

Comment:
Do we need this string to be separate? @lj-kpmg ?

Reply:
Not really at the moment, I assume on fares it's referred to in a few places so was put in here, we can get rid of it for the time being

Reply:
done